test/functionalTest/cases/1785_error_on_empty_attr_type/error_on_empty_attr_type.test

```diff
@@ -0,0 +1,141 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I guess the issue was solved by the error logic refactor I recently did. Thus, I'd suggest to add the corresponding entry to CNR about #1875 fixing.

(To be 100% precise, this .test should be tested agains Orion 1.1.0 and check it fails; however, maybe doing that is excesive)